.github/actions/move-released-issues-and-create-sync-pr-v1/README.md

```diff
@@ -30,6 +31,7 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           project-number: '66'
+          column-name: 'released'
```

Comment:
This change and the prior one in `label-and-move-released-issues` seem out-of-scope for this pr. Neither of the changes in the `about-release-candidate`or `release-candidate-ready-for-qa` actions use `move-released-issues-and-create-sync-pr` and add a `column-name` input.